cli/prisma2/src/Version.ts

```diff
@@ -60,7 +60,7 @@ export class Version implements Command {
   }
 }
 
-// Prisma CLI Version   : prisma2@2.0.0-alpha.473
+// Prisma CLI Version   : @prisma/cli@2.0.0-alpha.473
```

Comment:
Maybe ` @prisma/cli      : 2.0.0-alpha.473` instead?

Reply:
It a comment about the output of `prisma2 --version` so no

Reply:
Yes of course, this suggested a clearer output as `@prisma/cli` now already communicates that this is the CLI

Reply:
Done

Reply:
Hope you did not just do that because I suggested it, but because it is actually better ;) This was more a random idea...